src/hooks/useRHHCConfig.tsx

```diff
@@ -43,48 +39,16 @@ export default function useRHHCConfig(
       }
       return false;
     };
-    const getRoutedInternalHref: Config['utils']['getRoutedInternalHref'] = (
-      link,
-      type
-    ) => {
-      if (!link) {
-        return '#';
-      }
-
-      const uri = getUri(link, internalHrefOrigins, getIsHrefExternal);
-
-      if (type === ModuleItemTypeEnum.Article) {
-        return getLocalizedCmsItemUrl(
-          ROUTES.ARTICLES,
-          { slug: uri.replace(/^\//, '') },
-          locale
-        );
-      }
-      if (type === ModuleItemTypeEnum.Page) {
-        return getLocalizedCmsItemUrl(
-          ROUTES.PAGES,
-          { slug: uri.replace(/^\//, '') },
-          locale
-        );
-      }
-      if (type === ModuleItemTypeEnum.Event) {
-        return getLocalizedCmsItemUrl(
-          ROUTES.COURSES,
-          { eventId: uri.replace(/^\//, '') },
-          locale
-        );
-      }
-      //TODO: test the default case
-      return getLocalizedCmsItemUrl(link, {}, locale);
-    };
     return {
       ...rhhcDefaultConfig,
       components: {
         ...rhhcDefaultConfig.components,
         Head: (props) => <Head {...props} />,
         Link: ({ href, ...props }) => <Link href={href || ''} {...props} />,
-        EventCardContent: (props) => <EventDetails {...props} />,
-        ArticleCardContent: (props) => <ArticleDetails {...props} />,
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Could this work with type of `Record<string,unknown>` or just`unknown` ?

Reply:
Actually, I think they should be typed in the HCRC-lib. Some reason why the typing is not exported after the build done by the Rollup.

Reply:
But yeah, the unknown would be better than any with disabled linter. 👍

Reply:
https://thoughtbot.com/blog/typescript-stop-using-any-there-s-a-type-for-that